app/controllers/home_controller.rb

```diff
@@ -0,0 +1,16 @@
+class HomeController < ApplicationController
+
+  def index
+    @movies = Movie.all
+    @movies_sort = @movies.sort_by{ |movie| movie[:rank] }.reverse
+    @movies_ten = @movies_sort.first(10)
+
+    @books = Book.all
+    @books_sort = @books.sort_by{ |book| book[:rank] }.reverse
+    @books_ten = @books_sort.first(10)
+
+    @albums = Album.all
+    @albums_sort = @albums.sort_by{ |albums| albums[:rank] }.reverse
+    @albums_ten = @albums_sort.first(10)
```

Comment:
You have exposed several variables to the view layer that don't need to be here. In this case, since you're only displaying the first ten albums to the user, I would highly recommend using instance variables only for your `@movies_ten`, `@books_ten`, and `@albums_ten` variables.

This code looks very similar to the code used in your individual medium controllers. To DRY this up, you could think about using [scopes](http://guides.rubyonrails.org/active_record_querying.html#scopes) or another method in your model that contains this sorting & cropping. That way, instead of using a few lines to run these calculations each time in the controller, you could say right away `Album.ten_best` or `Album.ranked` (or however you call your scope / other method).